testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -327,29 +331,29 @@ impl TxEmitter {
                     e
                 )
             })?[0];
-        Ok(AccountData {
+        Ok(LocalAccount::new(
             address,
-            key_pair: self.mint_key_pair.clone(),
+            self.account_key(),
             sequence_number,
-        })
+        ))
     }
 
-    pub async fn load_diem_root_account(&self, client: &JsonRpcClient) -> Result<AccountData> {
+    pub async fn load_diem_root_account(&self, client: &JsonRpcClient) -> Result<LocalAccount> {
         self.load_account_with_mint_key(client, diem_root_address())
             .await
     }
 
-    pub async fn load_faucet_account(&self, client: &JsonRpcClient) -> Result<AccountData> {
+    pub async fn load_faucet_account(&self, client: &JsonRpcClient) -> Result<LocalAccount> {
         self.load_account_with_mint_key(client, testnet_dd_account_address())
             .await
     }
 
-    pub async fn load_tc_account(&self, client: &JsonRpcClient) -> Result<AccountData> {
+    pub async fn load_tc_account(&self, client: &JsonRpcClient) -> Result<LocalAccount> {
         self.load_account_with_mint_key(client, treasury_compliance_account_address())
             .await
     }
 
-    pub async fn load_dd_account(&self, client: &JsonRpcClient) -> Result<AccountData> {
+    pub async fn load_dd_account(&self, client: &JsonRpcClient) -> Result<LocalAccount> {
         let mint_key: Ed25519PrivateKey = generate_key::load_key(DD_KEY);
         let mint_key_pair: KeyPair<Ed25519PrivateKey, Ed25519PublicKey> = KeyPair::from(mint_key);
```

Comment:
Its odd to me that we load the key here but then don't actually use that key below when constructing the Account

Reply:
you are right, test with AOS DD account will fail. I used replace_all maybe mis-changed this one

Reply:
oh, no, I think we do not actually care about mint key for AOS account, but I think put corresponding key there makes more sense